docs/apache-airflow-providers-slack/operators/sql_to_slack.rst

```diff
@@ -0,0 +1,38 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+.. _howto/operator:SqlToSlackOperator:
+
+SqlToSlackOperator
+========================
+
+Use the :class:`~airflow.providers.slack.transfers.sql_to_slack` to post messages to predefined Slack
+channels.
```

Comment:
```suggestion
channel.
```
We don't support sending to more than 1 channel do we?

Reply:
Oh... 

Slack Incoming Webhook based on [Slack App](https://api.slack.com/messaging/webhooks) can't change channel,  in other word one webhook = one channel

Also exists legacy [Incoming Webhook](https://slack.com/apps/A0F7XDUAZ-incoming-webhooks?tab=more_info) based on Deprecated Slack Integration which can change channel.

The problem was it's unclear what exactly intend to use SlackWebhookHook. Actually it could use both of them, but I don't think we need to focus on legacy features.